### DISCUSSION TITLE
Is this still actively maintained?

### DISCUSSION THREAD

**Discussion by albarago** [loophole/cli#272] [General] [answered]
I used this a few times in the past and it worked wonderfully... albeit not persistently. For a few days now it appears to be down. Is this project still maintained? Thanks for all the work that's went into this.

**Comment by albarago** [comment, marked as answer]
For those wondering.. it appears that the downtime was temporary. As of this comment, the tunnels are working properly. Thanks

**Reply by Misfortuned700** [reply]
> For those wondering.. it appears that the downtime was temporary. As of this comment, the tunnels are working properly. Thanks

Do you use it as of now? It appears it's been in downtime again.
Edit: How long they last usually?

**Reply by awdev1** [reply]
I've seen them last a month before lol
